.github/workflows/release_pdb.yml

```diff
@@ -0,0 +1,67 @@
+name: Release XSPdb Jobs
+on:
+  push:
+    branches: [pdb]
+  pull_request:
+    branches: [pdb]
+jobs:
+  build-xsdev-image:
+    runs-on: bosc 
+    container: ghcr.io/openxiangshan/xspdb:build-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set Env 
+        run: |
+          echo "RTL_HOME=$GITHUB_WORKSPACE/build/rtl" >> $GITHUB_ENV 
+          echo "PDB_HOME=$GITHUB_WORKSPACE/build/xspdb" >> $GITHUB_ENV 
+          echo "PYTHON_VERSION=$(python3 --version | awk '{print $2}' | cut -d'.' -f1-2)" >> $GITHUB_ENV 
+          echo "PICKER_INCLUDE=$(picker --show_xcom_lib_location_cpp|grep include|awk '{print $2}')" >> $GITHUB_ENV
+          echo "pythonversion=$PYTHON_VERSION" >> $GITHUB_ENV 
+      - name: Init Xiangshan
+        run: make init
+      - name: Setup mill
+        uses: jodersky/setup-mill@v0.3.0
+        with:
+          mill-version: 0.12.3
+      
+      - name: Prepare XiangShan
+        run: | 
+          echo "NOOP_HOME=$GITHUB_WORKSPACE" >> $GITHUB_ENV 
+          NOOP_HOME=$GITHUB_WORKSPACE make sim-verilog WITH_CONSTANTIN=0 WITH_CHISELDB=0 JVM_XMX=16g
+      - name: Prepare Difftest
+        run: | 
+          cd $GITHUB_WORKSPACE/
+          NOOP_HOME=$GITHUB_WORKSPACE make -C difftest/ difftest_python WITH_CONSTANTIN=0 WITH_CHISELDB=0
+          cp $GITHUB_WORKSPACE/difftest/src/test/vsrc/common/* $RTL_HOME/
+          cp $PDB_HOME/python/_difftest.so $RTL_HOME/libdifftest.so 
+          cd $RTL_HOME/ 
+          ls | grep -E "\.sv$|\.v$|\.cpp$|\.so$" | grep -v "SimTop\.sv" > $RTL_HOME/picker.f || true 
```

Comment:
we should do all these works in a Makefile target, and use a simple `make pdb` command in CI steps.

this can make sure everyone can build a pdb binary in their own environment.